### ISSUE TITLE
Error when run npm start: Class constructor ServeCommand cannot be invoked without 'new'

### ISSUE THREAD

**Issue by jonizen** [glook/webpack4-react-boilerplate#7]
Hi!

Nice work, I tried it out and it gives me an error when running `npm run start` with the latest node and npm version (15.5.0) after looking at the error message `(node:50249) UnhandledPromiseRejectionWarning: TypeError: Class constructor ServeCommand cannot be invoked without 'new'` I looked at the webpack-cli repo and found [Error when run npm start: Class constructor ServeCommand cannot be invoked without 'new':](https://github.com/webpack/webpack-cli/issues/2272) which have reported the error. 

In short, the order needs to be: Syntax - `webpack [command] [options]` and the latest webpack needs to be 4.3.1.

Added a pull-request for the issue, tested and working #6 

Cheers!

**Comment by glook** [comment]
Hello @jonizen, sorry for long delay. Now you request was merged. Thank you!
